src/main/java/com/moodybluez/enterprise/controller/MoodyBluezController.java

```diff
@@ -38,6 +32,10 @@ public String showRegistrationForm(Model model) {
         return "index";
     }
 
+    @GetMapping("/aboutus")
+    public String aboutus() {
+        return "aboutus";
+    }
```

Comment:
Created an Endpoint which was being called through UI and since it was not created, it was throwing error, so added this.

Reply:
is there a matching page for aboutus.html?